pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -184,6 +186,50 @@ nodesLoop:
 	return res, nil
 }
 
+func (r *Runtime) GetGadgetInfo(ctx context.Context, desc gadgets.GadgetDesc, gadgetParams *params.Params, args []string) (*runTypes.GadgetInfo, error) {
+	ctx, cancelDial := context.WithTimeout(ctx, time.Second*ConnectTimeout)
+	defer cancelDial()
+
+	pods, err := getGadgetPods(ctx, []string{})
+	if err != nil {
+		return nil, fmt.Errorf("get gadget pods: %w", err)
+	}
+	if len(pods) == 0 {
+		return nil, fmt.Errorf("get gadget pods: Inspektor Gadget is not running on the requested node(s)")
```

Comment:
This is based on already existing code, also, it's highly probably that this PR will be merged after https://github.com/inspektor-gadget/inspektor-gadget/pull/2078, so this will go away.